CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 ## Prerelease
 
+### Updates
+
+- Updates Storybook to latest version.
```

Comment:
Put down the version number insteada of "latest version" because in future updates, those will also be "latest version".